### ISSUE TITLE
Value of "Out Of Date"

### ISSUE THREAD

**Issue by jeremejevs** [Jguer/yay#331]
*Skipped the template, since this isn't a bug.*

The "Out Of Date" message should be opt-in, in my opinion. This information is effectively nonactionable for me (and, I'd imagine, for the majority of Yay users): as much as I'd like to contribute, [the effort](https://wiki.archlinux.org/index.php/Arch_User_Repository#Foo_in_the_AUR_is_outdated.3B_what_should_I_do.3F) isn't worth it for some patch-version update. And if I'm truly missing something, which is available in a newer release, it's very likely that I'll become aware of it without Yay's help.

**Comment by Morganamilo** [comment]
You're right it's not really actionable, it's designed to be an informational warning, along with missing and orphaned packages.

Personally I don't see it worth the effort to make it a config option, it's one line of text. If this does get turned into a config option it will most definitely be enabled by default.

**Comment by jeremejevs** [comment]
It's not about the info, it's about the attention it grabs.

![](https://i.imgur.com/BjAi7Rd.png)

A whole lot of red and bold for something that's out of my hands :slightly_smiling_face: Maybe it could be demoted to a regular info-level message? Or printed only when the flag hasn't been previously seen? Or some sort of a combo of the two, where it's a warning at first, and then a message afterwards?

Not a big deal, of course, just reporting something that feels out of place.

**Comment by jeremejevs** [comment]
Well... missing and orphaned packages **are** actionable, so the boldness is appropriate for them, it's good to notice them. "Out Of Date" is different.

**Comment by Morganamilo** [comment]
Maybe they are actionable maybe they're not, it would depend on the user/package, I get missing package warnings because I have some custom pkgbuilds that are not in the AUR. I guess I could make a local repo to fix the warning but really I just live with it. When it comes to orphaned packages I would guess only a small amount of users know how to maintain packages so for them it would be unactionable.

Also the three warnings come from the same function call so in my eyes I see them as the exact same thing. None of these are really severe either.

I think it's best to come up with a consistent warning structure and use that for all the warnings.

**Comment by jeremejevs** [comment]
Please disregard my last comment. They aren't actionable, of course (by my definition). I read "orphaned" and immediately thought about the pacman type of "orphan", not AUR.

I agree that all three could be toned down equally.

**Comment by Morganamilo** [comment]
I've been working on a lot of formatting tweaks all around Yay, here's what I have for the upgrade menu.

Old:
![image](https://user-images.githubusercontent.com/16593899/38620309-10176362-3d96-11e8-8f7f-ba3518be5057.png)

New:
![image](https://user-images.githubusercontent.com/16593899/38620331-1e313586-3d96-11e8-8577-4cf4dd6658e2.png)


What you think so far?

**Comment by AlexWayfer** [comment]
> This information is effectively nonactionable for me (and, I'd imagine, for the majority of Yay users)

Just for note: sometimes it actionable for me. For example, we had `atom-editor` and `atom-editor-bin` in the AUR. And I'm not sure, which is updated more often. So, if I see, that installed `atom-editor-bin` has not been updated for a long time, and I want the new release — I can check the version of another, `atom-editor` package. If it has newer version — I can replace out-of-date `atom-editor-bin` with `atom-editor`. And vice versa. 

The same for `telegram-desktop`.

For now we have non-`bin` versions of these packages in the community repo, but it can occurs with other packages.
